src/widgets/survey/ui/Intermediate.tsx

```diff
@@ -197,19 +204,28 @@ function Intermediate({
       activityId: activityId,
       executionGroupKey,
       userIdentifier,
-      startTime: getActivityStartAt(progressRecord)!,
-      endTime: Date.now(),
+      startTime: getUnixTimestamp(getActivityStartAt(progressRecord)!),
+      endTime: getUnixTimestamp(Date.now()),
```

Comment:
Please remove getUnixTimestamp from createdAt, startTime, endTime, and scheduledDate.

It has been fixed recently but just in case you may miss it when solving conflicts

Reply:
ok